Kernel/Memory/AnonymousVMObject.cpp

```diff
@@ -156,7 +156,8 @@ AnonymousVMObject::AnonymousVMObject(AnonymousVMObject const& other, NonnullRefP
     , m_shared_committed_cow_pages(move(shared_committed_cow_pages))
     , m_purgeable(other.m_purgeable)
 {
-    ensure_cow_map();
+    // FIXME: Propagate errors
+    MUST(try_ensure_cow_map());
```

Comment:
`ErrorOr<>::release_value_but_fixme_should_propagate_errors()`

Reply:
I think "you must do this" is a better name than "forcefully release the value that this function returns" for what we want to do here.

Reply:
In my opinion, code should document it self, and when the code says MUST that means that it shall never fail, until I read the comment, while the other one says what it is doing, release a value although there may be an error, which we should handle.

On the topic, also regarding the other remark,
That function is designed to be an eyesore.

Reply:
`MUST(..)` is designed for code paths which truly can't fail or code paths we will never or can't fix. 

That doesn't seem to be the case here? 

Ultimately either way works and this is a silly bike shed for holding up this PR IMHO.